data/battle_scripts_1.s

```diff
@@ -7876,10 +7876,9 @@ BattleScript_MegaEvolutionAfeterString:
 	waitmessage B_WAIT_TIME_LONG
 	setbyte gIsCriticalHit, 0
 	handlemegaevo BS_ATTACKER, 0
-	handlemegaevo BS_ATTACKER, 1
 	playanimation BS_ATTACKER, B_ANIM_MEGA_EVOLUTION
 	waitanimation
-	handlemegaevo BS_ATTACKER, 2
+	handlemegaevo BS_ATTACKER, 1
```

Comment:
Since you're already making changes to this battle script, may as well correct its label I guess, `BattleScript_MegaEvolutionAfeterString` -> `BattleScript_MegaEvolutionAfterString` 😆

Reply:
Done :)